<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResurrectPenaltyDialog</class>
 <widget class="QDialog" name="ResurrectPenaltyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resurrect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Choose your resurrection penalty</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="levelsRadioButton">
        <property name="text">
         <string>Pay with 10 levels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="skillsRadioButton">
        <property name="text">
         <string>Pay with 2.5% of skill points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="statsRadioButton">
        <property name="text">
         <string>Pay with 4% of stat points</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResurrectPenaltyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResurrectPenaltyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
